core/src/com/unciv/models/translations/Translations.kt

```diff
@@ -443,6 +452,8 @@ private fun String.translatePlaceholders(language: String, hideIcons: Boolean):
 private fun String.translateIndividualWord(language: String, hideIcons: Boolean): String {
     if (Stats.isStats(this)) return Stats.parse(this).toString()
 
+    toDoubleOrNull().also { if (it != null) return it.tr(language) }
```

Comment:
Please stop using the "also" format everywhere, instead this should be 2 lines

Reply:
resolved in 9099a6648ff7bb4651ed3ad5c54d13300e66a107